### ISSUE TITLE
List ordering & date added

### ISSUE THREAD

**Issue by ulotrichous** [aadl/arborcat#88]
I remember there was some issue with the date added? But I heard from a list superuser who is still hoping that the ability to reorder a list, and see when an item was added, will return.

**Comment by taleon** [comment]
Yeah, I was partway through adding a sort option from oldest to newest. I'll try to finish that up this weekend. We still need to do some cleanup on date added since missed ~20k records on that.

**Comment by taleon** [comment]
Newest to oldest sorting for lists is in. Still need to cleanup date added.
